libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestRequest.m

```diff
@@ -26,6 +26,7 @@
 #import "SFRestAPI+Internal.h"
 #import "SalesforceSDKConstants.h"
 #import "SFJsonUtils.h"
+#import "SalesforceSDKCore.h"
```

Comment:
Because stringByURLEncoding is defined in NSString+SFAdditions.h.
Should I import
```
#import <SalesforceSDKCore/NSString+SFAdditions.h>
```
instead?